src/web_app/js/call.js

```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-var Call = function(params) {
+var Call = function(params, doNotRequestMediaAndIceServers) {
```

Comment:
Is this called somewhere else than in https://github.com/webrtc/apprtc/blob/6d86424372f85b2e2295126e9ebbf0ddc7f6348c/src/web_app/js/loopback.js#L27 ? Won't that break then?